kolibri/core/assets/src/views/tab-button/index.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+
+  <li>
+    <button
```

Comment:
Yes one is a router-link while the other is just a button that fires a simple click event.